lib/Backend/GlobOpt.cpp

```diff
@@ -19069,6 +19107,30 @@ GlobOpt::OptHoistInvariant(
     IR::BailOutKind bailoutKind)
 {
     BasicBlock *landingPad = loop->landingPad;
+
+    IR::Opnd* src1 = instr->GetSrc1();
```

Comment:
Moving this to the beginning of method is for cleanup from Louis.

---
In reply to: [101555390](https://github.com/Microsoft/ChakraCore/pull/2538#discussion_r101555390) [](ancestors = 101555390)